docs/run-node/interact-node.md

```diff
@@ -0,0 +1,49 @@
+<!--
+order: 3
+-->
+
+# Interacting with the Node
+
+## Pre-requisite Readings
+
+- [Running a Node](./run-node.md) {prereq}
+
+## Via CLI
+
+Now that your chain is running, it is time to try sending tokens from the first account you created to a second account. In a new terminal window, start by running the following query command:
+
+```bash
+simd query account $(simd keys show my_validator -a) --chain-id my-test-chain
```

Comment:
would be good to use variables instead repeating the sub-commands:
```
export MY_VALIDATOR=$(....)

Reply:
Yeah I agree. I'd call it `MY_VALIDATOR_ADDRESS` for the sake of clarity. The gist of it is that shorter examples are better than longer ones.